templates/workspace_services/ohdsi/template_schema.json

```diff
@@ -0,0 +1,356 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://github.com/microsoft/AzureTRE/templates/workspace_services/ohdsi/template_schema.json",
+  "type": "object",
+  "title": "OHDSI Workspace Service",
+  "description": "Provides OHDSI within the workspace",
```

Comment:
Can we have default name, description and overview. Azure ML is a good example.

https://github.com/microsoft/AzureTRE/blob/e5c0c9a81473b6624f1170f893951016c48ee521/templates/workspace_services/azureml/template_schema.json#L9-L29